<launch>
	<!-- New Version of Launch File for Robocup 2016 -->
	
	  <node name="people_detection_node" pkg="people_detection" type="people_detection_node" output="screen">

    	<!-- Camera Intrinsic -->
    	<param name="rgb_intrinsic" type="string" value="525 0.0 319.5 0.0 525 239.5 0.0 0.0 1.0"/>

		<!-- Trained SVM file reference path from package-->
		<param name="ref_svm_path" type="string" value="/trainedLinearSVMForPeopleDetectionWithHOG.yaml"/>
		
		<!-- Range (from the camera front) in which we want to detect-->
		<param name="detect_range" type="double" value="3.5"/>

		<!-- Confidence Threshold -->
		<param name="min_confidence" type="double" value="-1.5"/>
		
		<!-- Person's Height -->
		<param name="min_height" type="double" value="1.3"/>
		<param name="max_height" type="double" value="2.00"/>

		<!-- Head Minimum Distance -->
		<param name="head_min_distance" type="double" value="0.2"/>

		<!-- ENABLE USER INTERFACE -->
		<param name="ui" type="boolean" value="true"/>

        <!-- Maximum range to track between frame -->
        <param name="track_distance" type="double" value="0.4"/>

        <!-- Frame Count Condition -->
		<param name="out_track_condition" type="int" value="5"/>
        <param name="get_in_condition" type="int" value="2"/>
        <param name="get_in_check" type="int" value="5"/>

        <!-- Track Algorithm {0:SINGLE_NEAREST_NEIGHBOR_TRACKER, 1:MULTI_NEAREST_NEIGHBOR_TRACKER, 2:KALMAN_TRACKER} -->
        <param name="track_algorithm" type="int" value="1"/>
        <!-- Frame Count Method {0:UPDATE_NORMAL, 1:UPDATE_WITH_FRAME_COUNT} -->
        <param name="frame_count_method" type="int" value="1"/>


       </node>

		

</launch>
